src/badges.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+
+import type { AchievementType } from 'brewskey.js-api';
+
+import BackOnTheBusSmall from './resources/badges/70x70/BackOnTheBus-70x70.png';
```

Comment:
As I understand, metrobuilder doesn't support dynamic import, so I have to import every image manually.